x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_field.scss

```diff
@@ -0,0 +1,19 @@
+.appSearchResultField {
+  font-size: $euiSize * .875; // 14px
+  line-height: 1.5;
+  display: grid;
+  grid-template-columns: calc(35% + #{$euiSize * 3.45} /*55px*/ ) 1.875 * $euiSize /*30px*/ 1fr;
```

Comment:
Just wondering, do we have to be so pixel precise about this? 🤷‍♀️ I'm wondering if we can just round up to `$euiSize * 4` for the first and or `$euiSize * 2` for the second.

Reply:
Yep, we can do that. I'll do a sweep of the CSS.

Reply:
68f9780

Reply:
Love it!